library/core/src/cell.rs

```diff
@@ -2222,6 +2226,7 @@ impl<T: ?Sized> SyncUnsafeCell<T> {
     /// This call borrows the `SyncUnsafeCell` mutably (at compile-time) which
     /// guarantees that we possess the only reference.
     #[inline]
+    #[cfg_attr(not(bootstrap), rustc_never_returns_null_ptr)]
     pub const fn get_mut(&mut self) -> &mut T {
```

Comment:
```suggestion
    pub const fn get_mut(&mut self) -> &mut T {
```
This function does not return a pointer.